From e1fe60ab6ae7f192ab511eb81a246026e37a3317 Mon Sep 17 00:00:00 2001
From: Axel Lin <axel.lin@ingics.com>
Date: Sun, 2 Dec 2012 23:08:19 +0800
Subject: [PATCH 13/14] clocksource: sunxi_timer: Add terminating entry for
 sunxi_timer_dt_ids

The of_device_id table is supposed to be zero-terminated.

Signed-off-by: Axel Lin <axel.lin@ingics.com>
Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
---
 drivers/clocksource/sunxi_timer.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/clocksource/sunxi_timer.c
+++ b/drivers/clocksource/sunxi_timer.c
@@ -101,6 +101,7 @@ static struct irqaction sunxi_timer_irq
 
 static struct of_device_id sunxi_timer_dt_ids[] = {
 	{ .compatible = "allwinner,sunxi-timer" },
+	{ }
 };
 
 static void __init sunxi_timer_init(void)
